### ISSUE TITLE
🕷️[BUG] Missing Weapon Textures

### ISSUE THREAD

**Issue by shoober420** [onnoj/DeusExEchelonRenderer#26]
* Textures are not drawn on the GEP gun, medkit, soda can, and possibly others.
* RTX Remix 0.5.4
* Echelon render 0.3.9
* Steam with Deus Exe patch
* Portal with RTX works

Windows 11 27695
i9 13900kf @5.8ghz all cores
NVIDIA RTX 4090 | 560.94

Video:
https://youtu.be/3bql70xyFvU?si=PRoWTDzv0g98Vpc5

**Comment by onnoj** [comment]
Thanks for the bug report! I haven't seen this bug happen for me; is the GEP gun always untextured for you?

(for my reference)
![image](https://github.com/user-attachments/assets/bbcd6e8c-ca18-4296-987f-564dbe333705)

**Comment by shoober420** [comment]
Yes, as soon as i switch to the GEP gun, its instantly grey. This goes for the medkit and soda can as well.
